Source/Core/DolphinQt2/Config/Graphics/GraphicsWidget.h

```diff
@@ -0,0 +1,39 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QHash>
+#include <QWidget>
+
+class QFormLayout;
+class QGroupBox;
+class QLabel;
+
+class GraphicsWidget : public QWidget
```

Comment:
I'm not a fan of this pattern which uses subclassing to share common helper functions. To me, subclassing means, you should be able to write generic logic that _consumes_ the base class, not generic logic that's used inside subclasses (and this isn't a mix-in).

Can the description box be turned into a standalone widget that each tab then adds at the bottom? That looks like it's basically how it's being used anyways. Maybe something like:

```c++
m_description_box = MakeDescriptionBox();
installEventFilter(m_description_box);
m_description_box->AddDescription(m_enable_wireframe, TR_WIREFRAME_DESCRIPTION);
```